.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: NodeJS with Grunt
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: |
+        yarn install
+        grunt lint
+        grunt test
```

If think this would make the logs easier to read:
```suggestion
    - run: yarn install
    - run: grunt lint
    - run: grunt test
```

This way we can easily see which command failed.

---

Good call!